demo-react/src/components/svg/Cursor.tsx

```diff
@@ -0,0 +1,41 @@
+import { SVGProps } from 'react';
+
+interface Props extends SVGProps<SVGSVGElement> {
```

Comment:
is `SVGSVGElement` correct here?

Reply:
YES! It's so weird but it is a thing apparently 😂

Reply:
yikes